src/Sentry.Unity/SentryUnityOptions.cs

```diff
@@ -294,39 +295,38 @@ internal string? DefaultUserId
 
     internal List<string> SdkIntegrationNames { get; set; } = new();
 
-    public SentryUnityOptions() : this(false, ApplicationAdapter.Instance) { }
-
-    internal SentryUnityOptions(bool isBuilding, IApplication application, ISentryUnityInfo? unityInfo = null) :
-        this(SentryMonoBehaviour.Instance, application, isBuilding, unityInfo)
+    public SentryUnityOptions() :
+        this(false, ApplicationAdapter.Instance, SentryPlatformServices.UnityInfo, SentryMonoBehaviour.Instance)
```

Comment:
we access `SentryPlatformServices.UnityInfo` here but also in other places where they take SentryOptions but also `ISentryUnityInfo`. This makes it a lot more confusing.

Can we then only get it from here?

Once you do `new Options` we read `SentryPlatformServices.UnityInfo` and if it's null, we throw. Or go into the mode where this doesn't exist if that's a thing. But from that point any access to UnityInfo is through options. If you find references of the getter for `SentryPlatformServices.UnityInfo` you'd only see tests and this contructor.